core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -429,6 +430,9 @@ private static boolean includeAppDep(ResolvedDependency appDep, Optional<Set<Art
         if (!appDep.isJar()) {
             return false;
         }
+        if (appDep.getGroupId().equals("org.graalvm.nativeimage") || appDep.getGroupId().equals("org.graalvm.sdk")) {
```

Comment:
Do we need to only do this for a specific GraalVM version or will this work for all versions we support?

Reply:
It should work for all versions.

* Tested with 23.0-dev in https://github.com/graalvm/mandrel/actions/runs/3269063301
* And being tested with 22.2 in this PR.
* 22.3 should be no different.

Reply:
Thanks!